src/libaktualizr/uptane/tuf.cc

```diff
@@ -135,6 +135,7 @@ Target::Target(std::string filename, EcuMap ecus, std::vector<Hash> hashes, uint
       correlation_id_(std::move(correlation_id)) {
   // sort hashes so that higher priority hash algorithm goes first
   std::sort(hashes_.begin(), hashes_.end(), [](const Hash &l, const Hash &r) { return l.type() < r.type(); });
+  type_ = "binary";
```

Comment:
Maybe, it is better to set to "unknown"? I also think that "BINARY" is capitalized, though you can never be sure. Should we use `.toLower()`?

Reply:
> Maybe, it is better to set to "unknown"?

Agreed, done.

> I also think that "BINARY" is capitalized, though you can never be sure. Should we use `.toLower()`?

We don't appear to use "BINARY"; the only comparison we use for Target types is with "OSTREE". I think for now the comparison is fine as is.